src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/ArtifactsSizeTests/centos.9-x64.txt

```diff
@@ -2007,71 +2010,22 @@ sdk/x.y.z/.version: 114
-sdk/x.y.z/Containers/tasks/netx.y/Microsoft.Extensions.Primitives.dll: 64000
-sdk/x.y.z/Containers/tasks/netx.y/Microsoft.NET.Build.Containers.deps.json: 37253
+sdk/x.y.z/Containers/tasks/netx.y/Microsoft.NET.Build.Containers.deps.json: 26500
 sdk/x.y.z/Containers/tasks/netx.y/Microsoft.NET.Build.Containers.dll: 535040
-sdk/x.y.z/Containers/tasks/netx.y/MSBuild.dll: 800768
-sdk/x.y.z/Containers/tasks/netx.y/Newtonsoft.Json.dll: 1843200
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Common.dll: 365568
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Configuration.dll: 596992
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.DependencyResolver.Core.dll: 299008
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Frameworks.dll: 338944
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.LibraryModel.dll: 141824
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Packaging.dll: 1693696
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.ProjectModel.dll: 785408
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Protocol.dll: 2043904
-sdk/x.y.z/Containers/tasks/netx.y/NuGet.Versioning.dll: 146944
-sdk/x.y.z/Containers/tasks/netx.y/pl/Microsoft.DotNet.Cli.Utils.resources.dll: 7680
 sdk/x.y.z/Containers/tasks/netx.y/pl/Microsoft.NET.Build.Containers.resources.dll: 17408
-sdk/x.y.z/Containers/tasks/netx.y/pl/System.CommandLine.resources.dll: 8704
-sdk/x.y.z/Containers/tasks/netx.y/pt-BR/Microsoft.DotNet.Cli.Utils.resources.dll: 7680
 sdk/x.y.z/Containers/tasks/netx.y/pt-BR/Microsoft.NET.Build.Containers.resources.dll: 16384
-sdk/x.y.z/Containers/tasks/netx.y/pt-BR/System.CommandLine.resources.dll: 8704
-sdk/x.y.z/Containers/tasks/netx.y/ru/Microsoft.DotNet.Cli.Utils.resources.dll: 8704
 sdk/x.y.z/Containers/tasks/netx.y/ru/Microsoft.NET.Build.Containers.resources.dll: 20480
-sdk/x.y.z/Containers/tasks/netx.y/ru/System.CommandLine.resources.dll: 9216
-sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Security.Cryptography.Pkcs.dll: 741376
-sdk/x.y.z/Containers/tasks/netx.y/System.CommandLine.dll: 337408
-sdk/x.y.z/Containers/tasks/netx.y/System.Security.Cryptography.Pkcs.dll: 621056
-sdk/x.y.z/Containers/tasks/netx.y/System.Security.Cryptography.ProtectedData.dll: 34304
-sdk/x.y.z/Containers/tasks/netx.y/tr/Microsoft.DotNet.Cli.Utils.resources.dll: 7680
 sdk/x.y.z/Containers/tasks/netx.y/tr/Microsoft.NET.Build.Containers.resources.dll: 16896
-sdk/x.y.z/Containers/tasks/netx.y/tr/System.CommandLine.resources.dll: 8704
 sdk/x.y.z/Containers/tasks/netx.y/Valleysoft.DockerCredsProvider.dll: 39936
-sdk/x.y.z/Containers/tasks/netx.y/zh-Hans/Microsoft.DotNet.Cli.Utils.resources.dll: 7168
 sdk/x.y.z/Containers/tasks/netx.y/zh-Hans/Microsoft.NET.Build.Containers.resources.dll: 15872
-sdk/x.y.z/Containers/tasks/netx.y/zh-Hans/System.CommandLine.resources.dll: 8704
-sdk/x.y.z/Containers/tasks/netx.y/zh-Hant/Microsoft.DotNet.Cli.Utils.resources.dll: 7168
 sdk/x.y.z/Containers/tasks/netx.y/zh-Hant/Microsoft.NET.Build.Containers.resources.dll: 15360
-sdk/x.y.z/Containers/tasks/netx.y/zh-Hant/System.CommandLine.resources.dll: 8704
```

Comment:
https://github.com/dotnet/sdk/pull/41740 (I think? The dependencies seem to match up).